spherical_geometry/graph.py

```diff
@@ -375,7 +376,7 @@ def _sanity_check(self, title, node_is_2=False):
         for edge in self._edges:
             for node in edge._nodes:
                 if edge not in node._edges or node not in self._nodes:
-                    raise ValueError
+                    raise MalformedPolygonError(msg)
```

Comment:
Shouldn't sanity check raise `SanityError`? 😉

Reply:
Do you mean `Insanity`?

Reply:
Even better! 👍

Reply:
@pllim Could you please shine some light as to why the tests are being _`Expected`_ for over 6 hours?